src/sourmash/tax/__main__.py

```diff
@@ -260,11 +269,34 @@ def genome(args):
         with FileOutputCSV(summary_outfile) as out_fp:
             tax_utils.write_classifications(classifications, out_fp, limit_float_decimals=limit_float)
 
+    # write summarized output in human-readable format
+    if "human" in args.output_format:
+        summary_outfile, limit_float = make_outfile(args.output_base, "human", output_dir=args.output_dir)
+
+        with FileOutput(summary_outfile) as out_fp:
+            tax_utils.write_human_summary(classifications, out_fp, args.rank or "species")
+
     if "krona" in args.output_format:
+        # classifications only at a single rank
+        assert len(classifications) == 1
+        if args.rank:
+            assert args.rank in classifications
+
         krona_outfile, limit_float = make_outfile(args.output_base, "krona", output_dir=args.output_dir)
         with FileOutputCSV(krona_outfile) as out_fp:
             tax_utils.write_krona(args.rank, krona_results, out_fp)
 
+    if "lineage_csv" in args.output_format:
+        # should only classify at a single rank
```

Comment:
oh - so lineage_csv also forces classification to a given rank?

Reply:
I found the other comment in my e-mail but can't find it here, so I'll answer here in - in e-mail, you write:

>ah, i see - can classify at any rank, but print classifications only if available at given rank - yes?

>This might be something we want to allow for krona output, too

yes, print classifications only if available at a given rank. I've updated the comments a bit in a83ff3b4; if this is something you think we should consider for krona, please create an issue!